### PR TITLE
Feature/fsa 1070 translation workflow enhancements

### PR DESCRIPTION
**Pull request by johangant**
This PR provides:

1. Adapted menu links to move content specific tabs (content reminders, pending translations) into the tasks section of Content. This makes the UI a little more consistent for editors.
2. A new module that allows language teams to be notified by email when a new node entity is created, or when an editor checks a checkbox on the node edit form. This avoids spamming the team with smaller updates.
3. The email sent to the team tells them what changed, how to translate it and, where revisions exist, how to view the changes. This is intended to save them time, accelerate their workflow and reduce dependencies on colleagues contacting them through other channels to communicate changes.

### CODE REVIEW

File: drupal/web/modules/custom/fsa_translation_notify/src/Form/FsaTranslationNotifySettings.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('fsa_translation_notify.settings');
+
+    $form['info'] = [
+      '#markup' => '<p>' . $this->t('Content translation notifications are 
+        sent to the email address below. A notification is always sent for any 
+        new content pages created, or when the \'Notify language team of this change\'
+        checkbox is checked when saving new changes.'),
```
Comment (paulmoloneywunder) [inline drupal/web/modules/custom/fsa_translation_notify/src/Form/FsaTranslationNotifySettings.php]: Missing a closing <p> tag.

File: drupal/web/modules/custom/fsa_custom/fsa_custom.module [hunk 2]
```diff
@@ -372,6 +372,32 @@ function fsa_custom_form_alter(&$form, $form_state, $form_id) {
...
+  if ($variables['view']->storage->id() != 'content' || $variables['field']->field != 'nothing') {
+    return;
+  }
+
+  $revisions = \Drupal::entityTypeManager()->getStorage('node')->revisionIds($variables['row']->_entity);
+  if (count($revisions) < 2) {
+    return;
+  }
+
+  // What it the language of the current row? We only want to show
```
Comment (paulmoloneywunder) [inline drupal/web/modules/custom/fsa_custom/fsa_custom.module]: Change "What it the..." to "What is the..."

**Comment by paulmoloneywunder** [comment]
Couple of very minor issues. Tested and works well!

Couple of extra notes:

- Perhaps we could add a message if the user tries to send an email when translation team email field is empty? Could help with debugging, but likely to be a minor issue.

- Diff link doesn’t show anything when edits are done in non-standard node elements (example I found was editing a paragraph). Not sure what change could be made here but worth explaining to those editing the site.

File: drupal/web/modules/custom/fsa_translation_notify/src/Form/FsaTranslationNotifySettings.php (hunk 1 above)
Comment (johangant) [reply drupal/web/modules/custom/fsa_translation_notify/src/Form/FsaTranslationNotifySettings.php]: Good spot, thanks!

File: drupal/web/modules/custom/fsa_custom/fsa_custom.module (hunk 2 above)
Comment (johangant) [reply drupal/web/modules/custom/fsa_custom/fsa_custom.module]: 👍

**Comment by johangant** [comment]
Thanks for reviewing, @paulmoloneywunder. Interesting, and a bit frustrating, about poor diff/paragraphs integration. There seems to be a d.o issue relating to this https://www.drupal.org/project/paragraphs/issues/2836010 which looks like it could help, but at face value, it needs further work to get it to pass existing tests. I'll check it out when I'm next able to.

I'll fix the typo and missing element tag you spotted in the meantime.

**Pull request closed without merging**